### PR TITLE
Inte 503 metadata import export

### PR DESCRIPTION
**Pull request by kr1s-z**
Hey Guys,

In the end i had to switch to Vue as i svelte didn't want to recognise `loaded` message. Any feedback welcome

Thanks

### CODE REVIEW

**Comment by olivierchatry** [comment]
Is it an issue with svelte compiler ? That sounds really weird ? what happens if you put the message handler in the index.html file of the app ?

**Comment by kr1s-z** [comment]
> Is it an issue with svelte compiler ? That sounds really weird ? what happens if you put the message handler in the index.html file of the app ?

I didn't tried it :(

Directive below worked but `loaded` message was already send, other messages works fine. I had no option to do tokenRefresh as i didn't have backendurl too.
<svelte:window on:message={onWindowMessage} />

Other option was addEventListner but svelte compiler complains there is no `window`. Added browser to only use it client side but then nothing happend :(

I'm having suspicion is down to sveltekit

**Comment by olivierchatry** [comment]
Might be, anything on the git of sveltekit ?

File: metadata-import-export/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Doc tab
```
Comment (PeterJBassett) [inline metadata-import-export/README.md]: Suggest changing the title to something like FieldTwin API Demo Metadata Import Export

File: metadata-import-export/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Description
+
+This tool is deployed as a tab inside FieldTwin Design. It shows how to respond to
+window messages, call the FieldTwin API, download and upload metadata using excel spreadsheet.
+
+This example queries FieldTwin for metadata attached to staggedAsset, connection or well using SheetJS library
```
Comment (PeterJBassett) [inline metadata-import-export/README.md]: Typo 'stagged' -> 'staged'

File: metadata-import-export/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+1. Self-hosted option: deploy the file `index.html` on a public or private web
+   hosting service and obtain the URL to reach it  
+   or  
+   Pre-hosted option: use the URL `https://xvisionas.github.io/FieldTwin-Integration-Demo/metadata-import-export/`
+2. In FieldTwin Admin, go to Account Settings, then Integrations, and click _Create New Tab_.
+   Set the following values:  
+   ```
+   Name:                           Docs Tab
+   URL:                            <the URL from step 1>
```
Comment (PeterJBassett) [inline metadata-import-export/README.md:22]: In my demo, I had a problem that FieldTwin only allows https URLs and I couldn't see a way to use a http://localhost running demo?

File: metadata-import-export/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+   Use GET verb:                   yes
+   Do not pass arguments in URL:   yes
+   Tab Display Position:           Module Panel (default)
+   ```
+
+## Usage
+
+1. Open a project in FieldTwin Design, and select _staggedAsset_,  _connection_,  _well_
```
Comment (PeterJBassett) [inline metadata-import-export/README.md]: typo stagged

**Review by PeterJBassett — APPROVED** [review]
Hi Kris, looks good, I just spotted some typos in the README, and a question.

**Review by fozcode — APPROVED** [review]

**Pull request merged**